terraform/create_tf_vars.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+KEYS_JSON=$(cf service-key tanf-keys deployer | grep -A4 "{")
+if [ -z "$KEYS_JSON" ]; then
+  echo "Unable to get service-keys, you may need to login to Cloud.gov first"
+  echo "Run cf login --sso and attempt to retry running this script"
+  exit 1
+fi
+
+# Requires installation of jq - https://stedolan.github.io/jq/download/
+CF_USERNAME_DEV=$(echo "$KEYS_JSON" | jq -r '.username')
+CF_PASSWORD_DEV=$(echo "$KEYS_JSON" | jq -r '.password')
+
+CF_SPACE="tanf-dev"
+
+touch variables.tfvars
```

Comment:
This puts an empty file in the root of the `terraform` directory in addition to the correct file under `/dev/`. I believe that the /dev/ one is correct, but the README also mentions it being at the root of the directory.

Reply:
Thanks, yeah the `>> ./dev/variables.tfvars` will create this file if it's missing so the `touch` isn't needed.